audb/core/publish.py

```diff
@@ -104,7 +104,31 @@ def _find_attachments(
     # release dependencies to removed attachment files
     db_attachment_files = []
     for attachment_id in list(db.attachments):
-        db_attachment_files += db.attachments[attachment_id].files
+        path = audeer.path(db.root, db.attachments[attachment_id].path)
+        if os.path.isdir(path):
+            folders = audeer.list_dir_names(
+                path,
+                recursive=True,
+                hidden=True,
+                basenames=True,
+            )
+            for folder in folders:
+                if not os.listdir(audeer.path(path, folder)):
```

Comment:
Us there a specific reason we use `os.listdir()` and not `audeer.list_file_names()`?

Reply:
True, and it's of course easier than calling `list_file_names()` and `list_dir_names()`. So I guess it makes sense to stick wtih `os.lisdir()` here.